gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/dataset/ConvertibleHiveDataset.java

```diff
@@ -215,6 +219,11 @@ private ConversionConfig(Config config, Table table, String destinationFormat) {
       this.destinationDataPath = resolveTemplate(config.getString(DESTINATION_DATA_PATH_KEY), table);
 
       // Optional
+      if (config.hasPath(DESTINATION_VIEW_KEY)) {
+        this.destinationViewName = Optional.of(resolveTemplate(config.getString(DESTINATION_VIEW_KEY), table));
```

Comment:
Didn't use it because resolveTemplate cannot support nulls, but added null support there